### PR TITLE
#6043 dark skeleton

### LINKED ISSUE #6043: Skeleton dark mode
It would be nice to make all the loading skeletons in dark mode darker
image
- right now its afaik the same as in light mode
- is it possible to alter the color settings a bit? 
image
- left is k-shade over the skeleton, the right is k-grey, something darker would be really nice.

Thanks!

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 \_\_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #6043
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/ksm/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-05-18 at 8 15 39 PM

### <samp>🤖 Generated by Copilot at 22f6db0</samp>

Added a background gradient animation to the NFT card loading placeholders using the theme variables. This change enhances the UI of the `NeoSkeleton` component.

### <samp>🤖 Generated by Copilot at 22f6db0</samp>

> _`ktheme` gradient_
> _animates the skeleton_
> _winter's placeholder_

### CODE REVIEW

File: libs/ui/src/components/NeoSkeleton/NeoSkeleton.scss [hunk 1]
```diff
@@ -13,11 +13,20 @@ $grey-lightest: hsl(0, 0%, 93%);
 $skeleton-margin: 0.25rem;
 $base-border-radius: 4px;
 .neo-skeleton {
-
   &-item {
     &.no-margin {
       margin: 0 !important;
     }
+
+    background-size: 400% 100%!important;
```
Comment (daiagi) [inline libs/ui/src/components/NeoSkeleton/NeoSkeleton.scss:21]: why is this needed?

Comment (prachi00) [reply libs/ui/src/components/NeoSkeleton/NeoSkeleton.scss:21]: it is getting overridden and we need it for animation

**Review by daiagi — APPROVED** [review]

**Comment by vikiival** [comment]
pay 25 usd

**Comment by prachi00** [comment]
@vikiival bot failed it seems

**Comment by prachi00** [comment]
@vikiival bot failed it seems

**Comment by yangwao** [comment]
@prachi00 can u update to DOT 
- https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md

**Comment by prachi00** [comment]
@yangwao updated the payout link in the pr template with dot address, can we try now? 

PS. should we update the payout bot link to new format with dot then? I can do that

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 5.26 USD/DOT ~ 5.703 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x653982ff10ab0151faee27870ba577581dcbc20cc33b0a91b7e35231bec571fb](https://polkadot.subscan.io/extrinsic/0x653982ff10ab0151faee27870ba577581dcbc20cc33b0a91b7e35231bec571fb)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
> PS. should we update the payout bot link to new format with dot then? I can do that

it's already made :) but if you spot where we have KSM, feel free to update!

**Pull request merged**